### PR TITLE
DX: Fix SCA findings

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

**Comment by SpacePossum** [comment]
failure is not related to the changes

File: src/Fixer/Alias/ModernizeStrposFixer.php [hunk 1]
```diff
@@ -163,6 +166,8 @@ private function fixCall(Tokens $tokens, int $functionIndex, array $operatorIndi
 
     /**
      * @param -1|1 $direction
+     *
+     * @return array{operator_index: int, operand_index: int}
```
Comment (julienfalque) [inline src/Fixer/Alias/ModernizeStrposFixer.php:170]: Not sure why this is not reported:
```suggestion
     * @return null|array{operator_index: int, operand_index: int}
```

**Review by julienfalque — APPROVED** [review]

**Comment by julienfalque** [comment]
Thank you @SpacePossum.

**Comment by SpacePossum** [comment]
thanks for patching up the PR and the merge @julienfalque  ! :)

File: src/Fixer/Alias/ModernizeStrposFixer.php (hunk 1 above)
Comment (SpacePossum) [reply src/Fixer/Alias/ModernizeStrposFixer.php]: I agree, great catch, not sure why SCA didn't trip

**Pull request merged**